fs/operations/rc.go

```diff
@@ -214,7 +214,12 @@ func rcSingleCommand(ctx context.Context, in rc.Params, name string, noRemote bo
 		if err != nil {
 			return nil, err
 		}
-		_, err = CopyURL(ctx, f, remote, url)
+		autoFilename, err := in.GetBool("autoFilename")
+		if err != nil {
```

Comment:
This makes the parameter required - we want to allow it to be missing for backwards compatibility, so this should be OK.

```
autoFilename, _ := in.GetBool("autoFilename")
```

Reply:
Fixed.